integration/ct_prep_test.sh

```diff
@@ -25,9 +25,9 @@ echo "Provisioning test log (Tree ID: 0) in database"
 # Default to one RPC server and one HTTP server.
 RPC_SERVER_COUNT=${1:-1}
 HTTP_SERVER_COUNT=${2:-1}
-BASE_RPC_PORT=36961
-BASE_HTTP_PORT=6961
-LB_PORT=46962
+BASE_RPC_PORT=36962
```

Comment:
Ah, I just happened to have something else listening on 46962 on my local machine.